Signal/src/ViewControllers/MessageDetailViewController.swift

```diff
@@ -751,4 +752,18 @@ class MessageDetailViewController: OWSViewController, UIScrollViewDelegate {
 
         updateTextLayout()
     }
+
+    // MARK: MediaDetailPresenter
+
+    public func presentDetails(mediaMessageView: MediaMessageView, fromView: UIView) {
```

Comment:
This method was added during merge resolution. This is how we avoid passing the attachmentStream/viewItem to the mediaMessageView.